Overview.src.html

```diff
@@ -1492,21 +1506,21 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
 
 <h3>Referrer policies</h3>
 
-<p>A <dfn title=concept-referrer-policy>referrer policy</dfn> is the empty string,
+<p>A <dfn title=concept-referrer-policy data-export>referrer policy</dfn> is the empty string,
 "<code>no-referrer</code>", "<code>no-referrer-when-downgrade</code>", "<code>same-origin</code>",
 "<code>origin</code>", "<code>strict-origin</code>", "<code>origin-when-cross-origin</code>",
 "<code>strict-origin-when-cross-origin</code>", or "<code>unsafe-url</code>".
 
-<pre class=idl>enum <dfn>ReferrerPolicy</dfn> {
+<pre class=idl>enum <dfn data-dfn-type=enum data-export>ReferrerPolicy</dfn> {
   "",
-  "no-referrer",
-  "no-referrer-when-downgrade",
-  "same-origin",
-  "origin",
-  "strict-origin",
-  "origin-when-cross-origin",
-  "strict-origin-when-cross-origin",
-  "unsafe-url"
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer-when-downgrade</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>same-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>strict-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>origin-when-cross-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>strict-origin-when-cross-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>unsafe-url</dfn>"
 };</pre>
```

Comment:
Fair I guess. Though there has been some talk of moving this into Referrer Policy, I'm not objecting to taking this in for now.